pandas/tests/scalar/test_nat.py

```diff
@@ -348,3 +348,16 @@ def test_to_numpy_alias():
     result = NaT.to_numpy()
 
     assert isna(expected) and isna(result)
+
+
+@pytest.mark.parametrize("op", [
```

Comment:
rather than this can you use create a fixture similar to all_compare_operators but w/o the == and != ops (see pandas.conftest) and use that.